src/main/java/de/rabea/game/Game.java

```diff
@@ -1,64 +1,54 @@
 package de.rabea.game;
 
-import de.rabea.player.GuiPlayer;
-import de.rabea.player.UnbeatableComputerPlayer;
-
 public class Game {
 
     private final UserInterface userInterface;
-    private final Player player;
-    private final Player opponent ;
+    private final Player player1;
+    private final Player player2;
     private Player currentPlayer;
+    private Player notCurrentPlayer;
 
-    public Game(UserInterface userInterface, Player player, Player opponent) {
+    public Game(UserInterface userInterface, Player player1, Player player2) {
         this.userInterface = userInterface;
-        this.player = player;
-        this.opponent = opponent;
-        this.currentPlayer = player;
+        this.player1 = player1;
+        this.player2 = player2;
+        this.currentPlayer = player1;
+        this.notCurrentPlayer = player2;
```

Comment:
Yes, that's why I introduced it. So I can pass the correct player to the Gui because by the time I call the `finishGame()` method, the players have already been switched again. Previously I had an if-statement in the `play()` method, so that the mark was only switched  when the game was over. But feedback was that the play method was too complex, which is why I took it out... 

Alternatives I'm seeing:
- pass the wrong mark to the `userInterface.announceGameEnd()` method and then switch inside that method. But that's bad because it's duplication and I have to remember in all classes that implement the userInterface interface to switch the mark back. And why would I pass something that's wrong in the first place?
- adding the if statement into the `switchPlayer()` method. It means that it's still there in the `Game` class but at least the player is only switched when the game is still running. I'll do that.